to_do_list_part_two.py

```diff
@@ -1,7 +1,21 @@
 def add_task(task_list):
-    add_your_task = input("Enter the task:    ")
-    task_list.append(add_your_task)
-    print(f"{add_your_task} has been added to the list.")
+    task = input("Enter the task: ")
+
+    priority = input("Enter the priority(high, medium, low): ")
+    if priority != "high" or "medium" or "low":
+
+        print("Enter the correct word in high,medium,low")
+        return
+
+    deadline = input("Enter the dead line(YYYY-MM-DD): ")
```

Comment:
Can you handle invalid inputs?
Such as entering an incorrect date format like `2024/10/15`